src/libcalamaresui/utils/Paste.cpp

```diff
@@ -40,11 +40,15 @@ logFileContents()
         return QByteArray();
     }
     QFileInfo fi( pasteSourceFile );
-    if ( fi.size() > 16_KiB )
+    sizeLimit = ( sizeLimit < 0 ) ? 1024*1024 : sizeLimit * 1024;            //For KiB to bytes
+    cDebug() << "Log upload size limit was set to " << sizeLimit << " bytes";
+    if ( fi.size() > sizeLimit and sizeLimit > 0 )
     {
-        pasteSourceFile.seek( fi.size() - 16_KiB );
+        // Fixme : this following line is not getting pasted
+        cDebug() << "Only last " << sizeLimit << " bytes of log file (" << fi.size() << ") uploaded" ;
```

Comment:
You don't need extra spaces between items; `std::cout` does not introduce spaces between items, but QDebug does (by default, although there is a NoQuote item you can send to turn that off).